src/ImageSharp/ImageFrameCollection.cs

```diff
@@ -143,7 +143,7 @@ public Image<TPixel> CloneFrame(int index)
         /// <inheritdoc/>
         public ImageFrame<TPixel> CreateFrame()
```

Comment:
Let's create an overload for this that accepts a color, then we cover all bases.

Reply:
I see the method implements a member from interface IImageFrameCollection, wouldn't it be nice to change it to:
```
CreateFrame(TPixel backgroundColor = default)
```

Reply:
I think we don’t actually need the interface. I can’t see why someone would need to implement it themselves.

Reply:
Yeah, always choose overloads.